Server/JokesOnYou.Web.Api/Services/UserService.cs

```diff
@@ -25,7 +25,7 @@ public async Task DeleteUser(string id)
 
         public async Task<IEnumerable<User>> GetAll()
         {
-            throw new NotImplementedException();
+            return await _userRepository.GetUsersAsync();
```

Comment:
Returning a full user model would not be good.
Use a DTO (Data Transfer Object) and only return the information that you want to return.
We don't want the receiver of this list to get hashed passwords.